snomed/com.b2international.snowowl.snomed.datastore.server/src/com/b2international/snowowl/datastore/server/snomed/version/SnomedPublishManager.java

```diff
@@ -75,7 +81,9 @@
 public class SnomedPublishManager extends PublishManager {
 
 	private Set<String> componentIdsToPublish = newHashSet();
-	private Collection<SnomedModuleDependencyRefSetMember> newModuleDependencyRefSetMembers;
+	
+	// Map<Pair<moduleId, referencedComponentId>, Uuid> 
+	private Collection<Pair<String, String>> changedModuleDependencyRefSetMembers;
```

Comment:
This collection contains rather module and referenced component ids than actual refset members. Please rename it to reflect the purpose better!